AuditEventProcessor.cpp

```diff
@@ -122,13 +129,159 @@ void AuditEventProcessor::static_callback(void *au, dummy_enum_t cb_event_type,
     processor->callback(au);
 }
 
+bool AuditEventProcessor::process_execve()
+{
+    static const std::unordered_set<std::string> execve_fields = { "arch", "syscall", "success", "exit", "items", "ppid", "pid", "auid", "uid", "gid", "euid", "suid", "fsuid", "egid", "sgid", "fsgid", "tty", "ses", "comm", "exe", "key", "name", "inode", "dev", "mode", "ouid", "ogid", "rdev", "nametype", "cwd", "cmdline" };
+
+    if (_num_records < 5 || auparse_get_type(_state) != AUDIT_SYSCALL) {
+        return false;
+    }
+
+    const char *syscall = auparse_find_field(_state, "syscall");
+    auparse_first_field(_state);
+
+    if (syscall == nullptr || atoi(syscall) != SYS_execve) {
```

Comment:
The execve syscall number is different between 32bit (11) and 64bit (59). Please use auparse_interp_field() and then compare with "execve". Also, we should probably add support for the "execveat" syscall.